client/cmdhfmf.c

```diff
@@ -335,11 +375,47 @@ int usage_hf14_nack(void) {
 	PrintAndLog("Options:");
 	PrintAndLog("       h             this help");
 	PrintAndLog("       v             verbose");
-	PrintAndLog("samples:");
+	PrintAndLog("Examples:");
 	PrintAndLog("       hf mf nack");
 	return 0;
 }
 
+int GetHFMF14AUID(uint8_t *uid, int *uidlen) {
+	UsbCommand c = {CMD_READER_ISO_14443a, {ISO14A_CONNECT | ISO14A_NO_DISCONNECT, 0, 0}};
+	clearCommandBuffer();
+	SendCommand(&c);
+	UsbCommand resp;
+	if (!WaitForResponseTimeout(CMD_ACK, &resp, 2500)) {
+		//if (!silent) 
+		PrintAndLog("iso14443a card select failed");
+		DropField();
+		return 0;
+	}
+	
+	iso14a_card_select_t card;
+	memcpy(&card, (iso14a_card_select_t *)resp.d.asBytes, sizeof(iso14a_card_select_t));
+	memcpy(uid, card.uid, card.uidlen * sizeof(uint8_t));
+	*uidlen=card.uidlen;
+	return 1;

```

Comment:
you need to change, so device disconnects afterwards.  remove  ISO14A_NO_DISCONNECT